### DISCUSSION TITLE
2024/11/16/%E4%BD%BF%E7%94%A8%20Github%20Action%20%E8%87%AA%E5%8A%A8%E5%8C%96%E9%83%A8%E7%BD%B2%20Hexo%20%E5%8D%9A%E5%AE%A2/

### DISCUSSION THREAD

**Discussion by giscus** [bingxin666/blogmessages#7] [Announcements]
# 2024/11/16/%E4%BD%BF%E7%94%A8%20Github%20Action%20%E8%87%AA%E5%8A%A8%E5%8C%96%E9%83%A8%E7%BD%B2%20Hexo%20%E5%8D%9A%E5%AE%A2/

恰巧发现我的博客是基于我自己写的 Action 流部署后推送到另一个仓库的，虽然现在已经有了更好的替代品，但还是分享出来作为参考。

https://blog.byteloid.one/2024/11/16/%E4%BD%BF%E7%94%A8%20Github%20Action%20%E8%87%AA%E5%8A%A8%E5%8C%96%E9%83%A8%E7%BD%B2%20Hexo%20%E5%8D%9A%E5%AE%A2/

**Comment by a5mimnc2** [comment]
bro 你的服务器是在哪里购买的

**Reply by bingxin666** [reply]
蛮杂......
不过基本上都是小厂，大厂太贵了买不起.jpg

**Comment by kmizmal** [comment]
使用peaceiris/actions-gh-pages@v4不再需要Github Token，[参考](https://github.com/kmizmal/kmizmal.github.io/blob/main/.github/workflows/deploy.yml)。连构建一起解决了，方便~~跑路~~备份
